ecosystem/opUSDC/op-usdc.md

```diff
@@ -0,0 +1,112 @@
+# Purpose
+
+This document proposes a solution for implementing the [bridged USDC standard](https://www.circle.com/blog/bridged-usdc-standard) that can be easily upgraded to native USDC. This solution accomplishes the Circle requirements and accommodates OP Stack specifications to allow Chain Operators to perform an easy and secure migration in agreement with Circle.
+
+# Summary
+
+OpUSDC is a modular and efficient reference implementation that extends USDC across the Optimism ecosystem and OP Stack implementations. By leveraging the existing message-passing system and dedicated adapters, USDC can be deployed from day one and seamlessly migrate to native USDC in the future.
+
+This implementation of OpUSDC for the OP Stack allows for:
+
+- Deploy and set up OpUSDC with a single transaction.
+- Accomplish with the Bridged USDC standard.
+- Leverage from the OP Stack security model used.
+- Enabling a chain operator to coordinate with Circle to upgrade into native USDC.
+
+In the end, the goal is to provide a secure pathway for Circle's participation in any OP Chain ecosystem, which favors [Cross-Chain Transfer Protocol (CCTP)](https://www.notion.so/d464d146ce244502af760a9b9e4765f4?pvs=21) integration and [other benefits](https://www.circle.com/blog/bridged-usdc-standard) once the migration is completed.
```

Comment:
Unable to access notion doc.

Reply:
sorry about that, should have a working link now